src/utils/ParseRouteUtils.js

```diff
@@ -660,6 +660,6 @@ function parseBusRoutes(busRoutes: Array<Object>, originName: string, destinatio
 
 export default {
   condenseRoute,
-  parseBusRoutes,
+  parseRoutes,
```

Comment:
wait so just so im understanding this correctly, parseBusRoutes got renamed to parseRoutes bc it could potentially have walking routes, but you still have the parseWalkingRoute in the case that ghopper doesn't give a walking route for a short distance?

Reply:
yep it's not guaranteed that ghopper will return both walking and bus routes